dist/index.html

```diff
@@ -1,9 +1,11 @@
 <!DOCTYPE html>
```

Comment:
In general we don't want to include the build files in the git repo as they get regenerated every time we do a build, so we need to do a couple of things:

* delete the dist folder and commit that change
* add `/dist` to the bottom of the `.gitignore` file

once you've done that these files should disappear from the PR